### ISSUE TITLE
ok.ru site gets Unable to extract player error

### ISSUE THREAD

**Issue by masterz5398** [ytdl-org/youtube-dl#31778]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2021.12.17**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [ ] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-F', 'https://ok.ru/videoembed/4289246857824', '--verbose']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.10.9 (CPython) - Windows-10-10.0.22621-SP0
[debug] exe versions: ffmpeg 2021-07-11-git-79ebdbb9b9-full_build-www.gyan.dev, ffprobe 2021-07-11-git-79ebdbb9b9-full_build-www.gyan.dev
[debug] Proxy map: {}
[Odnoklassniki] 4289246857824: Downloading webpage
ERROR: Unable to extract player; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "A:\windows\python requirements\py 3_10\lib\site-packages\youtube_dl\YoutubeDL.py", line 815, in wrapper
    return func(self, *args, **kwargs)
  File "A:\windows\python requirements\py 3_10\lib\site-packages\youtube_dl\YoutubeDL.py", line 836, in __extract_info
    ie_result = ie.extract(url)
  File "A:\windows\python requirements\py 3_10\lib\site-packages\youtube_dl\extractor\common.py", line 534, in extract
    ie_result = self._real_extract(url)
  File "A:\windows\python requirements\py 3_10\lib\site-packages\youtube_dl\extractor\odnoklassniki.py", line 149, in _real_extract
    unescapeHTML(self._search_regex(
  File "A:\windows\python requirements\py 3_10\lib\site-packages\youtube_dl\extractor\common.py", line 1012, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract player; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

## Description

it keeps showing unable to extract error and after hundreds of retries(as run command is in while loop) it works miraculously
i do os.system("youtube-dl -o file url")

**Comment by Vangelis66** [comment]
Running latest "daily" build (compiled from `master-git-20230303-g3e92c60`) seems to work OK here: 
```shell
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--ffmpeg-location', '.\\FFmpeg', '--external-downloader-args', '-v 8 -stats', '-vF', 'https://ok.ru/videoembed/4289246857824']
[debug] Encodings: locale cp1253, fs mbcs, out cp737, pref cp1253
[debug] youtube-dl version 2023.03.06.19419
[debug] Python version 3.4.4 (CPython) - Windows-Vista-6.0.6003-SP2
[debug] exe versions: ffmpeg n5.2-dev-2245-N-109649-gab8cde6, ffprobe n5.2-dev-2245-N-109649-gab8cde6, phantomjs 2.1.1, rtmpdump 2.4
[debug] Proxy map: {}
[Odnoklassniki] 4289246857824: Downloading webpage
[Odnoklassniki] 4289246857824: Downloading m3u8 information
[info] Available formats for 4289246857824:
format code  extension  resolution note
mobile       mp4        unknown
mpd-0        mp4        256x108    DASH video  104k , mp4_dash container, avc1.64000B, 23fps, mp4a.40.2 (48000Hz)
hls-195      mp4        256x108     195k , 23.0fps
lowest       mp4        unknown
mpd-1        mp4        426x180    DASH video  297k , mp4_dash container, avc1.64000D, 23fps, mp4a.40.2 (48000Hz)
hls-376      mp4        426x180     376k , 23.0fps
low          mp4        unknown
mpd-2        mp4        640x272    DASH video  625k , mp4_dash container, avc1.640015, 23fps, mp4a.40.2 (48000Hz)
hls-753      mp4        640x272     753k , 23.0fps
sd           mp4        unknown
mpd-3        mp4        852x362    DASH video 1115k , mp4_dash container, avc1.64001E, 23fps, mp4a.40.2 (48000Hz)
hls-1224     mp4        852x362    1224k , 23.0fps
hd           mp4        unknown
mpd-4        mp4        1280x544   DASH video 2478k , mp4_dash container, avc1.64001F, 23fps, mp4a.40.2 (48000Hz)
hls-2605     mp4        1280x544   2605k , 23.0fps
full         mp4        unknown
mpd-5        mp4        1920x818   DASH video 4274k , mp4_dash container, avc1.640028, 23fps, mp4a.40.2 (48000Hz)
hls-4431     mp4        1920x818   4431k , 23.0fps (best)
```
So, WFM...

>  - [ ] I've searched the bugtracker for similar issues including closed ones

Why haven't you? It appears this is a duplicate of #26745 (and possibly of `can't reproduce` #27006), which dates from 2020... 
Is, perhaps, [this comment](https://github.com/ytdl-org/youtube-dl/issues/26745#issuecomment-710129106) relevant in your case?

**Comment by dirkf** [comment]
Also: #30839.
